README.md

```diff
@@ -117,7 +116,7 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The second command sets up a new rails app using postgreSQL as the database mgmt system. Otherwise, it will default to mySQL (I think).
```

Comment:
Yes, also the first creates a new folder called "tunr" and installs files inside it, whereas the second installs the files inside the present working directory.